spatialos-sdk-tools/src/download_sdk/main.rs

```diff
@@ -20,7 +20,7 @@ fn main() {
 
     download_and_unpack(
         &SpatialPackageSource::WorkerSdk,
-        "c-static-x86_64-msvc_md-win32",
+        "c-static-x86_64-msvc_mt-win32",
```

Comment:
I switched to the Win32 lib which statically links the standard library. This avoids the need for everyone to install the MSVC 2015 redistributable all the time.